examples/pytorch/semantic-segmentation/run_semantic_segmentation.py

```diff
@@ -388,6 +388,7 @@ def compute_metrics(eval_pred):
         cache_dir=model_args.cache_dir,
         revision=model_args.model_revision,
         use_auth_token=True if model_args.use_auth_token else None,
+        ignore_mismatched_sizes=True,
```

Comment:
Does it cause RuntimeError if it's False?

Reply:
no :)) it works fine